source/modules/MEI.xml

```diff
@@ -3071,6 +3071,31 @@
       </rng:data>
     </content>
   </macroSpec>
+  <macroSpec ident="data.PGFUNC" module="MEI" type="dt">
+    <desc xml:lang="en">Page header and footer function; a value that defines the function (i.e., the placement) of the header or the footer.</desc>
+    <content>
+      <valList type="closed">
+        <valItem ident="all">
+          <desc xml:lang="en">Header or footer for all the pages, including first and last unless provided.</desc>
+        </valItem>
+        <valItem ident="first">
+          <desc xml:lang="en">Header or footer for the first page only.</desc>
+        </valItem>
+        <valItem ident="last">
+          <desc xml:lang="en">Header or footer for the last page only.</desc>
+        </valItem>
+        <valItem ident="alt1">
+          <desc xml:lang="en">The first of an alternating pattern of headers or footers.</desc>
+        </valItem>
+        <valItem ident="alt2">
+          <desc xml:lang="en">The second of an alternating pattern of headers or footers.</desc>
+        </valItem>
+      </valList>
+    </content>
+    <remarks xml:lang="en">
+      <p>An alternating pattern with "alt1" and "alt2" starts from the first page. However, if header of footer with a func="first" is also defined, it will shift the pattern by one page. A header or footer with func="last" will interupt the pattern accordingly.</p>
```

Comment:
```suggestion
      <p>An alternating pattern with "alt1" and "alt2" starts from the first page. However, if header or footer with a func="first" is also defined, it will shift the pattern by one page. A header or footer with func="last" will interupt the pattern accordingly.</p>
```